assets/scss/_explorer.scss

```diff
@@ -1,9 +1,48 @@
 .explorer {
 
-  &.pixelsets {
+  aside.sidebar-left {
+    @include xy-cell($size: 2.5, $gutter-output: false);
+
+    .filters {
+
+      form {
```

Comment:
we could have a class for that, and avoid HTML elements as much as possible.

Reply:
Yup. Sounds safer.